### ISSUE TITLE
NullPointerException in PassphraseDialogActivity.java:105

### ISSUE THREAD

**Issue by dschuermann** [open-keychain/open-keychain#1266]
From G Play

"openkeychain stürzt nach update bei Aufruf durch k9 ab"

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{org.sufficientlysecure.keychain/org.sufficientlysecure.keychain.ui.PassphraseDialogActivity}: java.lang.NullPointerException
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2339)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2389)
at android.app.ActivityThread.access$600(ActivityThread.java:153)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1269)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:137)
at android.app.ActivityThread.main(ActivityThread.java:5289)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:525)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:739)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:555)
at dalvik.system.NativeStart.main(Native Method)
Caused by: java.lang.NullPointerException
at org.sufficientlysecure.keychain.ui.PassphraseDialogActivity.onCreate(PassphraseDialogActivity.java:105)
at android.app.Activity.performCreate(Activity.java:5133)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1087)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2293)
... 11 more
```

**Comment by Valodim** [comment]
missing requiredinput parcelable extra. the only place where the pendingintent for this is created is getRequiredInputPendingIntent in OpenPgpService, but that method accesses an attribute of the object so it is definitely not null from there

**Comment by dschuermann** [comment]
subkey id in requiredinput is null.

**Comment by dschuermann** [comment]
more user comments from G Play:

"just created a new letter in k9 mail. ticked on "sign" and inputted some text in the textfield. then pressed "send" button"

"simply selecting pgp encryption in Conversations led to OpenKeychain crash."

"Calling openkeychain from K-9 it crashes"

**Comment by Valodim** [comment]
should be fixed in 18844a20bba68f8a6bf89c2faaa698c403ff1c18
